internal/alb/tg/targets.go

```diff
@@ -64,6 +64,12 @@ func (c *targetsController) Reconcile(ctx context.Context, t *Targets) error {
 	if err != nil {
 		return err
 	}
+	if t.TargetType != elbv2.TargetTypeEnumInstance {
+		err = aws.PopulateTargetAZ(ctx, c.cloud, desired)
```

Comment:
can we move the PopulateTargetAZ to be member func of this TargetsController?
It doesn't fits into elbv2 since it involves multiple aws services.